SpotlightDownloader/ImageEncoder.cs

```diff
@@ -12,7 +12,7 @@ namespace SharpTools
     /// Jpeg part of this class is written by Pratik and davidwroxy from StackOverflow
     /// </remarks>
     /// </summary>
-    public static class ImageEncoder
```

Comment:
If I recall correctly, this class is a companion for the Windows 7 lockscreen code, and can be safely removed once the Windows 7 code is removed.